package.json

```diff
@@ -21,6 +21,13 @@
     "api:hooks": "node src/expert/hooks/server/index.js",
     "eslint:hooks": "eslint 'src/expert/hooks/**/*.(ts|tsx)'"
   },
+  "browserslist": [
+    "last 3 and_chr versions",
+    "last 3 chrome versions",
+    "last 3 opera versions",
+    "last 3 ios_saf versions",
+    "last 3 safari versions"
```

Comment:
Probably quite irrelevant in this context, but shouldn't we also add Firefox to the browserlist here?

Reply:
`and_chr` and `ios_saf` are related to mobile, I don't think they are usefull

Reply:
You're both totally right ! Thank you for your review.
I updated this PR.